projects/dsp-ui/design-documentation.md

```diff
@@ -132,6 +132,32 @@ The value component's selector is chosen from `valueTypeOrClass`. Since all valu
 Value components may have additional specific inputs for configuration that can be handled in `DisplayEditComponent`'s template as well
 ,e.g., additional configuration of how do display a date.
 
+#### Integration of CKEditor
+
+To edit XML, the viewer module relies on CKEditor. `TextValueAsXMLComponent` integrates the CKEditor library for Angular.
```

Comment:
do we want to leave `To edit XML`? as this is an HTML editor, it could be turned into an XML if we kept the source button